stix/idl/processing/vis/stix_l1_read.pro

```diff
@@ -38,11 +38,11 @@ bkg_ratio = 1.1
 cfl_ratio = 1.33
 triggers32=data.triggers(t2det-1)
 ;CFL and det8
-triggers32(8,*)=data.triggers(t2det(8))/(1.+cfl_ratio)*cfl_ratio
-triggers32(7,*)=data.triggers(t2det(7))/(1.+cfl_ratio)
+triggers32(8,*)=data.triggers(t2det(8)-1)/(1.+cfl_ratio)*cfl_ratio
+triggers32(7,*)=data.triggers(t2det(7)-1)/(1.+cfl_ratio)
 ;BKG and det 16
-triggers32(9,*)=data.triggers(t2det(9))/(1.+bkg_ratio)*bkg_ratio
-triggers32(15,*)=data.triggers(t2det(15))/(1.+bkg_ratio)
+triggers32(9,*)=data.triggers(t2det(9)-1)/(1.+bkg_ratio)*bkg_ratio
+triggers32(15,*)=data.triggers(t2det(15)-1)/(1.+bkg_ratio)

```

Comment:
I don't really follow what you're doing here. When we have the triggers per group then we can estimate the livetime for that detector directly.

Reply:
Do you mean to use the function 'stx_livetime_fraction' for computing the livetime?

Reply:
I just meant in general for the livetime - it is a single value per detector group so once you have the triggers for that group the livetime for both detectors is what you calculate.  I think the ratios are more for when we only have incomplete knowledge of the triggers.

Reply:
Ok, can we do this correction for the next release? I am planning to change the imaging pipeline using the same routines as for spectroscopy

Reply:
Yes I think that's a good plan.